btf/btf.go

```diff
@@ -470,8 +470,7 @@ func fixupDatasec(types []Type, sectionSizes map[string]uint32, offsets map[symb
 		case ".ksyms":
 			// .ksyms describes forward declarations of kfunc signatures.
 			// Nothing to fix up, all sizes and offsets are 0.
-			// Explicitly reject for now until we officially support kfuncs.
-			return fmt.Errorf("reference to %s: %w", name, ErrNotSupported)
+			continue
```

Comment:
Should we check there that the section only contains funcs?

Reply:
I now check the Datasec here and assume there will only be `*btf.Func`s in there going forward.